src/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProvider.php

```diff
@@ -108,6 +110,29 @@ private function extractBasicData(ProductForEditing $productForEditing): array
         ];
     }
 
+    /**
+     * @param ProductForEditing $productForEditing
+     *
+     * @return array<string, mixed>
+     */
+    private function extractQuantityData(ProductForEditing $productForEditing): array
```

Comment:
Please add unit test

Reply:
@jolelievre mentioned He will introduce tests for this class in one of his PR's

Reply:
Yes you asked the same thing in my PR about SEO This is a valid request so I'll implement them in my PR to keep this one no too big (it already add tests on Providers which was never done until now)